.angulardoc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "repoId": "eeef0326-152b-4dee-bcd0-776e21b7c84e",
+  "lastSync": 0
+}
```

Comment:
This appears to be from a VSCode extension. Should probably stay on the client-machine, since it's not a configuration for any dev dependency.

Reply:
Removed and resolved in cf52a2b1f98c4507f33dd1b7171104636094b829.